ui/src/app/cohort-search/overview/overview.component.css

```diff
@@ -21,6 +21,10 @@
   border-bottom: none;
 }
 
+clr-icon[shape^=error-standard] {
```

Comment:
Is ^ a typo?

Reply:
I know it looks weird but that's the correct CSS syntax to target a specific attribute and value